gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/ActivityStack.groovy

```diff
@@ -0,0 +1,20 @@
+package com.novoda.gradle.command
+
+class ActivityStack extends AdbTask {
```

Comment:
Correct, if we need access to the stack the AdbTask is not ideal as tried
to explain in the PR description
Am 13.07.2015 16:23 schrieb "Volker Leck" <notifications@github.com>:

> In
> gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/ActivityStack.groovy
> <https://github.com/novoda/gradle-android-command-plugin/pull/85#discussion_r34467426>
> :
>
> > @@ -0,0 +1,20 @@
> > +package com.novoda.gradle.command
> > +
> > +class ActivityStack extends AdbTask {
>
> Hmm, seems this task does not do anything by default.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/novoda/gradle-android-command-plugin/pull/85/files#r34467426>
> .
>